english/invite-friends/camp_fire/ghost-stories/ghost-stories.md

```diff
@@ -8,3 +8,5 @@ Do you:
 [Become the hero and tell everyone that you are going to see where the scream comes from](check_source/check_source.md)
 
 [Scare yourself to death and ask someone to go check where the scream came from](scared/scared.md)
+
+[Suggest to your friends that you had better go to the next room together to find out what happened](check_source_team/check_source_team.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.